source/_integrations/diagnostics.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: Diagnostics
+description: The diagnostics integration can provide integration and device information for debugging purposes.
+ha_category:
+  - Other
```

Comment:
Remote Python Debugger is in the Utility category. I know these aren't the same things but they seem to be in a similar vein of troubleshooting no? Should they be in the same category then?

Reply:
Dunno, this is now in the same category as System Health.

Reply:
Being in the same as system health make sense 👍